### PR TITLE
Specifying when events happen for ReadModel Scenario

### LINKED ISSUE #257: Cannot specify DateTime for ReadModel Scenario's given
The [ReadModel Scenario](https://github.com/qandidate-labs/broadway/blob/c2939cdd68edf1f44cdaef3ce0288fd1250e18d1/src/Broadway/ReadModel/Testing/Scenario.php#L82) allows specifying when an event happened for the 'when' helper

However, this is not supported for the [given](https://github.com/qandidate-labs/broadway/blob/c2939cdd68edf1f44cdaef3ce0288fd1250e18d1/src/Broadway/ReadModel/Testing/Scenario.php#L68) method.

The issue we are facing is that we are using the DateTime from the DomainMessage in one of our projectors but we have no control over [the value being generated](https://github.com/qandidate-labs/broadway/blob/c2939cdd68edf1f44cdaef3ce0288fd1250e18d1/src/Broadway/ReadModel/Testing/Scenario.php#L106) by the scenario

### PR DESCRIPTION
**Pull request by fritsjanb**
Fixes #257

### CODE REVIEW

File: src/Broadway/ReadModel/Testing/Scenario.php [hunk 1]
```diff
@@ -60,6 +62,11 @@ public function withAggregateId($aggregateId)
         return $this;
     }
 
+    public function usingDateTime(DateTime $dateTime)
```
Comment (asm89) [inline src/Broadway/ReadModel/Testing/Scenario.php]: `withDateTime()` and return `$this`?

**Comment by asm89** [comment]
I'm conflicted about this solution, because:

- the datetime object will be the exact same object for each event (opposed to a new object each time)
- the timing for each events in the tests will also be the same

I understand that under most conditions the latter is already the case, but it's something we should solve instead of mask further.

**Comment by asm89** [comment]
lgtm

**Comment by mente** [comment]
Why not use the same approach as it was for `when()`?

**Pull request merged**